### PR TITLE
TELCODOCS-1607: Updated diagram and note

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
Update the existing NVIDIA Architecture diagram and add support note. These are very minor changes to the diagram. The added note is a footnote removed from the diagram converted to a note. 

Jira: https://issues.redhat.com/browse/TELCODOCS-1607

Version(s):  openshift-4.14z, openshift-4.13.z, openshift-4.15 

Link to docs preview: https://69114--ocpdocs-pr.netlify.app/openshift-enterprise/latest/architecture/nvidia-gpu-architecture-overview

SME review: @egallen 
QE review: @wabouhamad

### CODE REVIEW

**Comment by StephenJamesSmith** [comment]
@egallen @wabouhamad Please review and Ack. Thx!

**Comment by StephenJamesSmith** [comment]
@egallen @wabouhamad Please review and Ack (LGTM). Thx!

**Review by skrthomas — COMMENTED** [review]
Groovy. /lgtm

File: modules/nvidia-gpu-enablement.adoc [hunk 1]
```diff
@@ -8,5 +8,11 @@
...
 The following diagram shows how the GPU architecture is enabled for OpenShift:
 
+
 .NVIDIA GPU enablement
-image::349_OpenShift_NVIDIA_GPU_arch_0723.png[NVIDIA GPU enablement]
+image::512_OpenShift_NVIDIA_GPU_enablement_1223.png[NVIDIA GPU enablement]
+
+[NOTE]
+====
+MIG is only supported with A30, A100, A100X, A800, AX800. H100, H200, and H800.
```
Comment (kcarmichael08) [inline modules/nvidia-gpu-enablement.adoc:17]: should that be a comma after AX800?

Comment (StephenJamesSmith) [reply modules/nvidia-gpu-enablement.adoc]: Fixed.

**Pull request merged**